azure-pipelines.yml

```diff
@@ -640,20 +640,36 @@ jobs:
       ArtifactName: rpm-mariner$(tag)
 
 
-- job: BuildRpmPackageCentOS7
-  displayName: Build Rpm Package CentOS 7
+- job: BuildRpmPackages
+  displayName: Build Rpm Packages
   # Uncomment this line to disable this job on Pull Requests
   # condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
   pool:
     vmImage: 'ubuntu-20.04'
+  strategy:
+    matrix:
+      CentOS7:
+        image: centos7
+        tag: centos7
+        # Should be 'rpm-centos7', but we keep 'yum' for backward compatibility
+        artifact: yum
+      Fedora35:
+        image: fedora
+        tag: 35
+        artifact: rpm-fedora35
+      Fedora36:
+        image: fedora
+        tag: 36
+        artifact: rpm-fedora36
```

Comment:
As far as I know, the latest `rpmbuild` is pretty slow (the task can take 30min). Will using a matrix drag down the duration of this task?

Reply:
Fedora build takes 15min and the task will be delayed for several minutes. I think it's okay.